src/elements/content-explorer/ContentExplorer.js

```diff
@@ -140,6 +150,8 @@ class ContentExplorer extends Component<Props, State> {
 
     table: any;
 
+    contentExplorerRef: refType<HTMLElement>;
```

Comment:
I see in other places we define the flow type inline such as https://github.com/box/box-ui-elements/blob/15326176688e562e3bb4e7c3077f7d66be40f348/src/features/metadata-instance-editor/Instance.js#L427